### ISSUE TITLE
error when applying the CRD

### ISSUE THREAD

**Issue by cpanato** [minio/minio-operator#35]
when applying the updated crd in the cluster i got this error:

```
 kubectl apply -n minio-operator -f docs/minio-operator/minio-operator.yaml
error: error validating "docs/minio-operator/minio-operator.yaml": error validating data: ValidationError(CustomResourceDefinition.spec): unknown field "preserveUnknownFields" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinitionSpec; if you choose to ignore these errors, turn validation off with --validate=false
```

k8s cluster version: kindest/node:v1.13.4

**Comment by nitisht** [comment]
@cpanato you can either update the k8s cluster version or comment the CRD spec from `preserveUnknownFields` to whole of `openAPIV3Schema` section

Validation is only available from K8s 1.15 onwards (https://kubernetes.io/blog/2019/06/19/kubernetes-1-15-release-announcement/) as mentioned in pre-requisites: https://github.com/minio/minio-operator#prerequisites

**Comment by cpanato** [comment]
thanks!
